portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -44,7 +31,12 @@ public CommentsApiResponse(List<CommentsDatastore.Comment> comments) {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Integer commentsLimit = Integer.parseInt(request.getParameter("commentsLimit"));
+    String commentsLimitStr = (String) request.getParameter(Constants.COMMENTS_LIMIT_PARAMETER);
+    if (commentsLimitStr.isEmpty()) {
+      throw new IllegalArgumentException("comments limit should not be empty");
+    }
+
+    Integer commentsLimit = Integer.parseInt(commentsLimitStr);
```

Comment:
parseInt throws NumberFormatException do you want to catch this and return an api compatible response?